lib/datadog/tracing/contrib/extensions.rb

```diff
@@ -80,6 +80,7 @@ def configure(&block)
 
                 # integration.patch returns either true or a hash of details on why patching failed
                 patch_results = integration.patch
+                integration.patcher.patch_results = patch_results
```

Comment:
I think the Patcher should be responsible for storing its own result.
By that I mean that this method should populate `@patch_results`: https://github.com/DataDog/dd-trace-rb/blob/54c5e1ee69df3089bc9f73c4c6f01d5046b4f004/lib/datadog/tracing/contrib/patcher.rb#L33-L42